app/lib/features/chat_ng/actions/copy_message_action.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+Future<void> copyMessageAction(
```

Comment:
All these message actions along with other are moved entirely to  ```actions``` for legibility.